packages/backend/src/graphql/resolvers/liquidity.ts

```diff
@@ -520,7 +616,8 @@ export const createIncomingPaymentWithdrawal: MutationResolvers<ApolloContext>['
       'incomingPaymentService'
     )
     const incomingPayment = await incomingPaymentService.get({
-      id: incomingPaymentId
+      id: incomingPaymentId,
+      tenantId: ctx.isOperator ? undefined : ctx.tenant.id
     })
     const webhookService = await ctx.container.use('webhookService')
     const event = await webhookService.getLatestByResourceId({
```

Comment:
small thing, but we could put these two gets into a Promise.all, same down below.

Also, if we cannot find the `incomingPayment` we should return `UnknownIncomingPayment` instead of `InvalidId`